src/pages/settings/Subscription/RequestEarlyCancellationPage/index.tsx

```diff
@@ -0,0 +1,139 @@
+import React, {useMemo, useState} from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import FeedbackSurvey from '@components/FeedbackSurvey';
+import FixedFooter from '@components/FixedFooter';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as Report from '@userActions/Report';
+import type {CancellationType, FeedbackSurveyOptionID} from '@src/CONST';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function RequestEarlyCancellationPage() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    // TODO: replace this with NVP_PRIVATE_CANCELLATION_DETAILS.cancellationType
```

Comment:
Will this be added in future PR ?

Reply:
Yes, exactly. This PR is UI only.

Reply:
We've done the backend work for this though, haven't we? 😕

Reply:
I don't know, I was only asked to take care of the UI part of this (and this is what the ticket says). However, I will be happy to integrate it with the backend too, maybe in a separate PR, wdyt?

Reply:
Yeah, @rlinoz's PRs are done for https://github.com/Expensify/App/issues/43285 and https://github.com/Expensify/Expensify/issues/405463.

Reply:
For the frontend portion from the doc: 

>Clicking submit calls the new CancelBillingSubscriptionNewDot command. We’ll pass the user’s survey response to the backend using the parameter cancellationReason, and we’ll call it using API.write().
>
>The frontend will need to create a new Onyx type called NVP_PRIVATE_CANCELLATION_DETAILS. It’ll have this shape:

> ```
> type PrivateCancellationDetails = {
>     cancellationDate: string,
>     cancellationReason: string,
>     cancellationType: "automatic" | "none" | "manual",
>     note: string,
>     requestDate: string,
>     subscription: PrivateSubscription,
> };
> ```

Reply:
>However, I will be happy to integrate it with the backend too, maybe in a separate PR, wdyt?

We're going to release the subscription page real soon, so I'm a little wary of having this PR go to prod without being hooked up to the backend tbh. I'd rather ship that together here.

Reply:
I agree it makes more sense to have the API calls working in this PR.

Reply:
Alright, @JKobrynski can you do that here then please? Thanks!